src/application/common/commands-common.ts

```diff
@@ -8,6 +8,7 @@
 import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
+let Globalize = require("globalize");
```

Comment:
I have just found the [type definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/globalize/index.d.ts) for `globalize`, cannot we use that?

Does `npm i -S @types/globalize` solve this issue? If so, we could use ES6 imports.